src/insns/acperm_32bit.adoc

```diff
@@ -52,9 +52,9 @@ The rules from <<acperm_rules>> must be followed when removing permissions.
 |  4  (RV32 only) | <<x_perm>>   | <<r_perm>>
 |  5  (RV32 only) | <<w_perm>>   | not(<<c_perm>>) or <<lm_perm>>
 |  6  (RV32 only) | <<x_perm>>   | <<w_perm>> or <<c_perm>>
-|  7              | <<el_perm>>  | <<c_perm>> and <<r_perm>>
-|  8              | <<lm_perm>>  | <<c_perm>> and <<r_perm>>
-|  9              | <<sl_perm>>  | <<c_perm>>
+|  7              | <<el_perm>>  | <<c_perm>> and <<r_perm>> and <<lm_perm>>
+|  8              | <<lm_perm>>  | <<c_perm>> and <<r_perm>> and (<<w_perm>> or <<el_perm>>)
+|  9              | <<sl_perm>>  | (<<c_perm>> and <<lm_perm>> and (<<x_perm>> or <<w_perm>>)
```

Comment:
Do we really need to explicitly list LM here? I guess it's implied since SL depends on W, and W implies LM. Not opposed to keeping it since I imagine the checks will be optimized out but wondering if the text can be simplified.

Reply:
```suggestion
|  9  (RV32 only) | <<sl_perm>>  | (<<c_perm>> and <<lm_perm>> and (<<x_perm>> or <<w_perm>>)
```

Reply:
yes - RV32 only was missing - will push a new commit soon

Reply:
If you remove `LM` from the SL term in rule 9 there are 10 routes to get an invalid encoding, but they all end with the same permissions. 
| R | W | C | LM | SL | X | Rule |
|--------|--------|--------|--------|--------|--------|--------|
| 1 | 1 | 1 | 0 | 1 | 1 | After & |
| 1 | 0 | 1 | 0 | 1 | 1 | Rule 5 - clear W |
| 1 | 0 | 1 | 0 | 1 | 1 | Rule 9 - but no LM inclusion (no change) |
| 1 | 0 | 1 | 0 | 1 | 0 | Rule 10 - clear X |

This leaves us with `R, C, SL` which doesn't map to any table entry.  If you have `LM` as part of the rule, then you remove `SL` end at `R, C`.

Reply:
Rule 9 without `LM` does not clear `SL` because `X` is still HIGH, but `SL` needs to be aware of `X` because Quadrant 1, row 2/3 has  `W == 0` and `C, LM, EL == 1` and `SL == inf` while Quadrant 3, row 3 has identical values for `W, C, LM, EL` however `SL=0`.  So the only differentiator between these two cases is that in Quadrant 1 has `X`

Reply:
closing as Alex has approved the PR